docs/components/movement-sensor/gps/set-up-base-station.md

```diff
@@ -0,0 +1,239 @@
+
+- Default: Disabled.
+- Enable this option. Enter the following credentials:
+  - **Caster Host:** rtk2go.com
+  - **Caster Port:** 2101
+  - **Caster User:** test@test.com
+  - **Mount Point:** [name of mountpoint from your [RTK2GO Casting Service reservation](#reserve-a-rtk2go-casting-service)]
+  - **Mount Point PW:** [password from your [RTK2GO Casting Service reservation](#reserve-a-rtk2go-casting-service)]
+
+For example:
+
+{{< imgproc src="/components/movement-sensor/sparkfun-base-configuration.png" alt="Recommended Base Configuration." resize="900x" style="width:450px" >}}
+
+### Update WiFi configuration
+
+Select the **WiFi Configuration** dropdown on the RTK setup configure page.
+
+**Networks:** Enter credentials for up to 4 WiFi networks.
+The device tries all networks and uses the best network available.
+This is essential for NTRIP client/server configuration.
+
+**Configure Mode:** In **AP** mode, the device broadcasts as an access point called “RTK-Config”.
+In **WiFi** mode, the device connects to local WiFi and is configurable on the displayed IP address.
+
+- Select **WiFi**.
+
+Your WiFi configuration should look like this, with the examples replaced with your real values:
+
+{{< imgproc src="/components/movement-sensor/sparkfun-wifi-configuration.png" alt="Example WiFi Configuration." resize="1000x" style="width:500px" >}}
+
+### Update Profile configuration
+
+Select the **Profile Configuration** dropdown on the RTK setup configure page.
+
+Select a profile and then assign settings to that profile.
+At reset, your RTK device will use the selected profile.
+Profile changes are saved when you select a different profile or when you press **Save Configuration**.
+
+Your **Profile Configuration** should look like this:
+
+{{< imgproc src="/components/movement-sensor/sparkfun-profile-configuration.png" alt="Example Profile Configuration." resize="900x" style="width:450px" >}}
+
+### Save your configuration changes
+
+Select **Save Configuration** on the RTK setup configure page and wait for your configurations to take effect.
+
+## Start RTK2GO casting
+
+Using the reference kit's **MODE** button, switch the mode back to the default **Base**.
+Press the **MODE** button once to activate the menu.
+Press the button again to step through the available modes.
+Once highlighted, pause on **Base** to select this mode.
+
+Because you configured the base station to be in **Survey-in** mode in the [update Base configuration step](#update-base-configuration), it will take your base station up to six minutes to go into **Xmitting RTCM** mode.
+Then, after a few minutes it should go into **Casting** mode.
+Once you see **Casting** on the mode menu display screen, go to `rtk2go.com:2101` in your browser and find your mount point in the source table.
+
+## Next steps: configure an RTK movement sensor
+
+Provide this mount point as the `ntrip_url` attribute in your Viam machine config for the RTK-enabled models of movement sensor.
```

Comment:
@susmitaSanyal Is this accurate? will it be URL? a bit unsure-- if another step lmk!

Reply:
ntrip_url is `rtk2go.com:2101` , mount point would look something like "MTPT" or "VIAM_BASE". This is something the user determines in step 1

Reply:
So how does mount point work? What do you need it for? Sorry for the confusion @susmitaSanyal

Reply:
Sierra and I had a discussion in person about this.